rules/windows/defense_evasion_posh_assembly_load.toml

```diff
@@ -0,0 +1,77 @@
+[metadata]
+creation_date = "2021/10/15"
+maturity = "production"
+updated_date = "2021/10/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the use of Reflection.Assembly to load PEs and DLLs in memory in Powershell Scripts. Attackers use
+this method to load executables and DLLs without writing to the disk, bypassing security solutions.,
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Suspicious .NET Reflection via PowerShell"
+references = [
+    "https://docs.microsoft.com/en-us/dotnet/api/system.reflection.assembly.load",
+]
+risk_score = 73
+rule_id = "e26f042e-c590-4e82-8e05-41e81bd822ad"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.code:"4104" and 
```

Comment:
@w0rk3r  For this (and all rules using PS logging), can we convert from `event.code:4104` to `event.category:process`. We would even use the `event.action` in addition to the category if needed.

Reply:
Done